spec/spec_helper.rb

```diff
@@ -12,7 +12,7 @@
 end
 Capybara.javascript_driver = :poltergeist
 
-Capybara.default_max_wait_time = 10
+Capybara.default_max_wait_time = 30
```

Comment:
Were you seeing issues locally w/ this wait time?  Just wondering if we should only set it to this under CI (so that it fails faster locally).

Thoughts?